### ISSUE TITLE
Player Tracker track..

### ISSUE THREAD

**Issue by Facc** [jonatkins/ingress-intel-total-conversion#228]
hey i cant open a new pull request so i post here.
Here is it, my 'issue'.. isnt it a bit tricky?
![tricky](https://f.cloud.github.com/assets/3955848/450395/6332c44c-b290-11e2-9468-c9c3595564e4.png)

i was thinking about different colors for different player tracks. Is it too hard?

thank jonatkins for all

**Comment by Hackswell** [comment]
Yes.  #467, #226, and #228, all appear related.  All involve enhancements to [Player Tracker], which is awesome by the way.

Here's my $0.03:
* Colour needs to be something other than blue, pink, or green.  Pink looks too much like Cross-Link colour.  Blue and green are obvious.
* Player Tracker, since it is a DASHED line, should be on TOP of any existing portal links.  I cannot see the player tracker well if they were creating links/fields.  The player tracker is rendered first, then the link is rendered over top of it.

My suggestions on colors:
* Resistance: Purple
* Enlightenment: Yellow
Purple and yellow contrast against each other very well, and they are similar enough to the faction colour to logically know which team it is.  (Purple is part blue, and green is part yellow)

* I love the idea of clicking on a player's icon and having it "boldened".


Thank you for all your hard work on this project, developers!
